mkdocs/mkdocs.yml

```diff
@@ -53,9 +53,20 @@ plugins:
   - search
   - minify:
       minify_html: true
+  - macros:
+      # Replace {{ variable }} syntax with [[ variable ]] to avoid conflicts
+      j2_block_start_string: '[[%'
+      j2_block_end_string: '%]]'
+      j2_variable_start_string: '[['
+      j2_variable_end_string: ']]'
         
 # Customization
 extra:
+  # Google Analytics
```

Comment:
Apparently, the previous way of enabling Google Analytics is deprecated. This is the new version to do so. 

https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#configuration

Apparently Universal Analytics (`UA-*) is also being phased out...